docs/pages/versions/unversioned/sdk/clipboard.md

```diff
@@ -30,8 +30,7 @@ export default function App() {
   const [copiedText, setCopiedText] = React.useState('');
 
   const copyToClipboard = () => {
-    /* @info */ Clipboard.setString('hello world');/* @end */
-
+    /* @info */ Clipboard.setString('hello world'); /* @end */
```

Comment:
Shouldn't we use `setStringAsync` in the examples already?

Reply:
Yes, you're right. I need to update all docs examples.